docs/source/usecases/ec2offhours.rst

```diff
@@ -1,32 +1,46 @@
 EC2 - Offhours Support
 ======================
 
-- Offhours are based on current time of the instance
+Offhours are based on current time of the instance. Note, in this case you could tag an instance with the following two tags: StopAfterHours: off=(M-F,18);tz=est; and StartAfterHours: on=(M-F,8). This would have the instance turn off every weekday at 6pm NY time, and turn on every day at 8am California time (since if no tz is set, it uses the default which is pt). Note when custodian runs, if it's 6:00pm or 6:59 pm NY time, it will shut down the VM you tagged this way. The key is the hour integer on the NY clock matching 18. If custodian runs at 5:59pm or 7:00pm NY time, it won't shut down the VM. Same idea for starting.
```

Comment:
current time of the instance... more like current time on the host/system executing custodian.